### PR TITLE
 Fix some nullable annotations

### PR DESCRIPTION
**Pull request by safern**
cc: @dotnet/nullablefc

### CODE REVIEW

**Comment by stephentoub** [comment]
From API review to PR in, what, 3 minutes? :)

**Comment by safern** [comment]
> From API review to PR in, what, 3 minutes? :)

Learned from the expert on doing that 😄

File: src/System.Private.CoreLib/src/System/RuntimeHandles.cs [hunk 1]
```diff
@@ -70,11 +70,11 @@ internal static Type GetTypeHelper(Type typeStart, Type[]? genericArgs, IntPtr p
 
         public static bool operator ==(RuntimeTypeHandle left, object right) { return left.Equals(right); }
 
-        public static bool operator ==(object left, RuntimeTypeHandle right) { return right.Equals(left); }
+        public static bool operator ==(object? left, RuntimeTypeHandle right) { return right.Equals(left); }
 
         public static bool operator !=(RuntimeTypeHandle left, object right) { return !left.Equals(right); }
 
-        public static bool operator !=(object left, RuntimeTypeHandle right) { return !right.Equals(left); }
+        public static bool operator !=(object? left, RuntimeTypeHandle right) { return !right.Equals(left); }
```
Comment (stephentoub) [inline src/System.Private.CoreLib/src/System/RuntimeHandles.cs]: You also need question marks on `object? right` in the first == above and the first != above.

Comment (safern) [reply src/System.Private.CoreLib/src/System/RuntimeHandles.cs]: Argh. Good catch 😄

**Comment by mjsabby** [comment]
@safern did you intend to keep individual commits when checking this change in? If you could squash and merge in the future unless required for some reason, it keeps the history linear and clean.

**Comment by safern** [comment]
No this time it was a mistake. I had been working on coreclr with merge commits because we were working on a feature branch and somehow I forgot to change the default to be squash and merge and accidentally merged it with a merge commit. I realized when already merged :sad:

**Pull request merged**